### PR TITLE
Null safety migration

### PR DESCRIPTION
**Pull request by jtmuller5**

### CODE REVIEW

**Comment by laves** [comment]
Awesome, thanks so much @jtmuller5 ! I'm going to make a couple quick notes here about the PR, but it looks great overall.

File: binding/flutter/CHANGELOG.md [hunk 1]
```diff
@@ -17,4 +17,7 @@
 * Updated Voice Processor
 
 ## [1.9.9] - 2021-04-12
-* Flutter 2.0 upgrade
\ No newline at end of file
+* Flutter 2.0 upgrade
+
+## [2.0.0] - 2021-04-24
```
Comment (laves) [inline binding/flutter/CHANGELOG.md:22]: We can't upgrade out Porcupine version to 2.0.0 yet, I'd revert this and I can update it.

File: binding/flutter/pubspec.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: porcupine
 description: A Flutter plugin for Picovoice's Porcupine offline wake word engine.
-version: 1.9.9
+version: 2.0.0
```
Comment (laves) [inline binding/flutter/pubspec.yaml:3]: I'll take care of versioning

File: binding/flutter/pubspec.lock [hunk 3]
```diff
@@ -77,10 +77,12 @@ packages:
   flutter_voice_processor:
     dependency: "direct main"
     description:
-      name: flutter_voice_processor
-      url: "https://pub.dartlang.org"
-    source: hosted
-    version: "1.0.3"
+      path: "."
```
Comment (laves) [inline binding/flutter/pubspec.lock:80]: I'll take care of updating this file for our next release - thanks

File: binding/flutter/pubspec.yaml [hunk 4]
```diff
@@ -2,9 +2,10 @@ name: porcupine
 description: A Flutter plugin for Picovoice's Porcupine offline wake word engine.
```
Comment (laves) [inline binding/flutter/pubspec.yaml:2]: I'll take care of updating this file

File: binding/flutter/CHANGELOG.md (hunk 1 above)
Comment (jtmuller5) [reply binding/flutter/CHANGELOG.md]: Okay this is reverted

File: binding/flutter/pubspec.yaml (hunk 2 above)
Comment (jtmuller5) [reply binding/flutter/pubspec.yaml]: Done

**Pull request merged**